roles/bwc/tasks/rbac.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+  - name: Create BWC RBAC directories
+    become: yes
+    file:
+      path: "{{ item }}"
+      mode: "u=rwx,g=rx,o=rx"
+      owner: st2
+      group: st2
+      state: directory
+    with_items:
+      - /opt/stackstorm/rbac/assignments
+      - /opt/stackstorm/rbac/roles
+
+  - name: Copy default RBAC roles to /opt/stackstorm/rbac/roles directory
+    become: yes
+    template:
+      src: rbac_roles/roles.yml.j2
+      dest: /opt/stackstorm/rbac/roles/{{ item.name }}.yaml
```

Comment:
Changing any file in `/rbac/` dir should notify `reload bwc_rbac` trigger.

Make sure we run that thing for other tasks in this file.